specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/examples/Patches_ListByBuilderResource.json

```diff
@@ -0,0 +1,30 @@
+{
+  "operationId": "Patches_ListByBuilderResource",
+  "parameters": {
+    "api-version": "2023-05-02-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "builderName": "testBuilder"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/builders/testBuilder/patches/testPatch-25fe4b",
+            "name": "testPatch",
```

Comment:
nit: should this be `testPatch-25fe4b` to match the name of the patch resource in the above `id` property?

Reply:
sorry i think this one's a typo. I'll fix that in the next commit. It should be something like this:
`.../builders/testBuilder/patches/testPatch`